yt_dlp/extractor/dlf.py

```diff
@@ -0,0 +1,209 @@
+                    'ext': 'mp3',
+                    'duration': 187,
+                    'thumbnail': 'https://assets.deutschlandfunk.de/a595989d-1ed1-4a2e-8370-b64d7f11d757/512x512.jpg?t=1679173825412',
+                    'uploader': 'Deutschlandfunk',
+                    'series': 'Sport am Samstag',
+                    'channel': 'deutschlandfunk'
+                }
+            }, {
+                'info_dict': {
+                    'id': '47e1a096',
+                    'title': r're:Rückkehr Russlands im Fechten [-/] "Fassungslos, dass es einfach so passiert ist"',
+                    'url': r're:https://(?:download\.deutschlandfunk|ondemand-mp3\.dradio)\.de/file/dradio/2023/03/11/weltfechtverband_laesst_russen_und_belarussen_wieder_zu_dlf_20230311_1912_47e1a096\.mp3',
+                    'ext': 'mp3',
+                    'duration': 602,
+                    'thumbnail': 'https://assets.deutschlandfunk.de/da4c494a-21cc-48b4-9cc7-40e09fd442c2/512x512.jpg?t=1678562155770',
+                    'uploader': 'Deutschlandfunk',
+                    'series': 'Sport am Samstag',
+                    'channel': 'deutschlandfunk'
+                }
+            }, {
+                'info_dict': {
+                    'id': '5e55e8c9',
+                    'title': r're:Wiederzulassung von Russland und Belarus [-/] "Herumlavieren" des Fechter-Bundes sorgt für Unverständnis',
+                    'url': r're:https://(?:download\.deutschlandfunk|ondemand-mp3\.dradio)\.de/file/dradio/2023/03/18/fechten_unverstaendnis_und_wut_nach_wiederzulassung_dlf_20230318_1930_5e55e8c9\.mp3',
+                    'ext': 'mp3',
+                    'duration': 187,
+                    'thumbnail': 'https://assets.deutschlandfunk.de/a595989d-1ed1-4a2e-8370-b64d7f11d757/512x512.jpg?t=1679173825412',
+                    'uploader': 'Deutschlandfunk',
+                    'series': 'Sport am Samstag',
+                    'channel': 'deutschlandfunk'
+                }
+            }],
+            'params': {'skip_download': 'm3u8'}
+        },
+        # Podcast feed with tag buttons
+        {
+            'url': 'https://www.deutschlandfunk.de/corso-100.html',
+            'info_dict': {
+                'id': False,
+                'title': r're:Kunst & Pop [-/] Corso',
+                'description': 'md5:ba0f80303683591bff4b7459077bef58'
+            },
+            'playlist_mincount': 34,
+            'params': {'skip_download': True}
+        },
+        # Podcast feed with no description
+        {
+            'url': 'https://www.deutschlandfunk.de/podcast-tolle-idee-100.html',
+            'info_dict': {
+                'id': False,
+                'title': 'Wissenschaftspodcast - Tolle Idee! - Was wurde daraus?',
+                'description': None
```

Comment:
Keys with `None` values can be omitted from tests

Let the test_download script generate the info_dicts for you by only manually entering `url` and `id`, and then `ext` (if single media) or `playlist_count`/`playlist_mincount` (if playlist). The test will fail and output the rest of the dict for you to copy/paste